io/net/virt-net/network_virtualization.py

```diff
@@ -545,6 +569,16 @@ def find_device_id(self):
                                           shell=True).strip()
         return device_id
 
+    def find_virtual_slot(self, dev_id):
+        """
+        finds the virtual slot for the given virtual ID
+        """
+        output = process.system_output("lsslot", ignore_status=True,
+                                       shell=True, sudo=True)
+        for slot in output.split('\n'):
+            if dev_id in slot:
+                return slot.split(' ')[0]
```

Comment:
Please handle the case when this slot is not got.
Either here or the function call.